Rotary encoder requires a module which cannot be installed

@Saiyato 
When I do volumio plugin install I get this error:

npm ERR! notarget No compatible version found: onoff-rotary@^0.2.1

This should be fixed (use another module or link the module as library)

https://github.com/volumio/volumio-plugins/blob/fc328b85e40c56a6554ef6ff8af10a86e9b2555e/plugins/user_interface/rotaryencoder/package.json#L37

--- a/plugins/user_interface/rotaryencoder/package.json
+++ b/plugins/user_interface/rotaryencoder/package.json
@@ -34,7 +34,7 @@
 		"net": "^1.0.2",
 		"once": "^1.4.0",
 		"onoff": "^1.1.9",
-		"onoff-rotary": "^0.2.1",
+		"onoff-rotary": "https://github.com/Saiyato/onoff-rotary/tarball/ebfbc16a0806762b2e08ab40a539f0fcd3302506",
 		"path-is-absolute": "^1.0.1",
 		"rimraf": "^2.6.2",
 		"slide": "^1.1.6",

